Module 7/ex1/main.cpp

```diff
@@ -0,0 +1,29 @@
+#include <iostream>
+
+template <typename T> void iter(T *array, size_t length, void (*func)(T &)) {

```

Comment:
Here you have as argument a pointer to a function taking an argument of type T.
How could you change it so that this "function pointer type" is defined outside of the function scope, and therefore reusable in other parts of the code ?

Reply:
it is called alias